data/andre_estimates_21_02.txt

```diff
@@ -0,0 +1,38 @@
+1	3	30	42	42	9	2

```

Comment:
I am not sure that we want to necessarily hold the raw data and then do the manipulation in the book. We can just process this once and be done with it

Reply:
We might want to use it with the age-groups later - also I think it is a nice bridge with the original paper.